README.md

```diff
@@ -4,51 +4,63 @@ You might want the convenient `cargo public-items` wrapper for this library. See
 
 # public_items
 
-List public items (the public API) of a Rust library crate by analyzing the rustdoc JSON of the crate. Enables diffing public API between releases and commits.
```

Comment:
Not sure why the readme shows a diff here, maybe enough if you merge origin/main into your branch

Reply:
Yes that helped.